skbio/metadata/__init__.py

```diff
@@ -0,0 +1,9 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
```

Comment:
Can you follow the structure described [here](http://scikit-bio.org/docs/latest/development/new_module.html) for adding a new subpackage to scikit-bio? It's okay to leave the docstring out since there's nothing public to document yet.